pallets/automation-time/src/lib.rs

```diff
@@ -633,7 +687,13 @@ pub mod pallet {
 			}
 			// Update the missed queue
 			let mut missed_queue = Self::get_missed_queue();
-			missed_queue.append(&mut missed_tasks);
+			for missed_task in missed_tasks {
+				let new_missed_task: MissedTask<T> = MissedTask::<T> {
+					task_id: missed_task,
+					execution_time: last_missed_slot,
```

Comment:
this is not the correct time for all of the missed_tasks

Reply:
ahh crap, yah i missed this. but yah, this should be the right date: https://github.com/OAK-Foundation/OAK-blockchain/pull/128/files/20ee59cd998b39757533d248a51ff5605dc1508f#diff-e6a13f7032a383e19b77d20043ca46041d94f5e49396812e2de1b2000c301e59R711